config/initializers/rswag_api.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+Rswag::Api.configure do |c|
+  # Specify a root folder where Swagger JSON files are located
+  # This is used by the Swagger middleware to serve requests for API descriptions
+  # NOTE: If you're using rswag-specs to generate Swagger, you'll need to ensure
+  # that it's configured to generate files in the same folder
+  c.swagger_root = Rails.root.to_s + '/doc'
```

Comment:
```suggestion
  c.swagger_root = "#{Rails.root}/doc"
```

I think this is the same